Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1673,6 +1674,35 @@ def lhegensim(fragment,howMuch):
     if k2 in PUDataSets:
         upgradeStepDict['RecoFull_trackingOnlyPU'][k]=merge([PUDataSets[k2],upgradeStepDict['RecoFull_trackingOnly'][k]])
 
+    upgradeStepDict['RecoFullGlobal'][k] = {'-s':'RAW2DIGI,L1Reco,RECO',
+                                      '--conditions':gt,
+                                      '--datatier':'GEN-SIM-RECO',
+                                      '-n':'10',
+                                      '--runUnscheduled':'',
+                                      '--eventcontent':'FEVTDEBUGHLT',
+                                      '--geometry' : geom
+                                      }
+    if cust!=None : upgradeStepDict['RecoFullGlobal'][k]['--customise']=cust
+    if era is not None: upgradeStepDict['RecoFullGlobal'][k]['--era']=era
+
+    if k2 in PUDataSets:
+        upgradeStepDict['RecoFullGlobalPU'][k]=merge([PUDataSets[k2],upgradeStepDict['RecoFullGlobal'][k]])
+
+    upgradeStepDict['RecoFullGlobal_trackingOnlyVal'][k] = {'-s':'RAW2DIGI,L1Reco,RECO,VALIDATION:@trackingOnlyValidation,DQM:@trackingOnlyDQM',
```

Comment:
I'm curious what would you name this once more than trackingOnly is added to the validation (e.g. trackingAndJetAndBlahAndBlah and so on)?
It may be better to have a common name